src/style.css

```
@@ -0,0 +1,92 @@
+body {
+  margin: 0;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+}
+
+.column {
+  display: flex;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  height: 300px;
+  width: 300px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: white;
+  flex-grow: 1;
+}
+
+.c-1 {
+  background-color: red;
+}
+
+.c-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.c-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.c-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.c-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.c-6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 899px) {
+  .row {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .column {
+    flex-basis: calc(100% / 3);
+  }
+  
+  .c-4 {
+    order: 3;
+  }
+  
+  .c-5 {
+    order: 2;
+  }
+  
+  .c-6 {
+    order: 1;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
```

```suggestion
@media (min-width: 600px) {
```
Поставь этот медиазапрос выше `min-width: 900`, и тогда можно будет удалить из `min-width: 900` все что касается flex-direction и flex-wrap